wordpress-mobile/WordPress-Android
# Use a webview for WP.com login

Fixes a fairly common login issue reported in https://a8c.slack.com/archives/C02AVAR9B/p1728576885606729.

-----

## To Test:
- Install both Jetpack and WordPress. 
- Try logging into WP.com
- Note that it works

-----

## Regression Notes

1. Potential unintended areas of impact
Could be other issues around login – I tried very narrowly address the issue in this PR – nothing has been removed, so everything should work like it did before (for instance, use re-authentication).

2. What I did to test those areas of impact (or what existing automated tests I relied on)
n/a

3. What automated tests I added (or what prevented me from doing so)
There wouldn't be a lot of benefit to automated tests at this point.

-----

## PR Submission Checklist:


-----

## Testing Checklist (strike-out the not-applying and unnecessary ones):
